src/modules/csm/csm.py

```diff
@@ -119,16 +119,20 @@ def build_report(self, blockstamp: ReferenceBlockStamp) -> tuple:
                 shares[no_id] += amount
 
         tree = self.make_tree(shares)
-        cid: CID | None = None
+        tree_cid: CID | None = None
```

Comment:
Can we split build_report function into smaller function

Reply:
Did my best.